src/test/java/bio/terra/service/snapshotbuilder/SnapshotRequestDaoTest.java

```diff
@@ -41,17 +40,21 @@ class SnapshotRequestDaoTest {
   @Autowired private DaoOperations daoOperations;
   @Autowired private SnapshotRequestDao snapshotRequestDao;
 
-  private Dataset dataset;
-  private Snapshot snapshot;
+  private Snapshot sourceSnapshot;
+  private Snapshot createdSnapshot;
   private SnapshotAccessRequest snapshotAccessRequest;
 
   private static final String EMAIL = "user@gmail.com";
+  private static final String FLIGHT_ID = "flight_id";
 
   @BeforeEach
   void beforeEach() throws IOException {
-    dataset = daoOperations.createDataset(DaoOperations.DATASET_MINIMAL);
```

Comment:
What is the reason not to save the dataset outside the scope of `beforeEach`?

Reply:
it was a sonar recommendation, because it was only used in this one place